x-pack/plugins/canvas/public/components/workpad_header/share_menu/flyout/flyout.component.tsx

```diff
@@ -88,11 +116,39 @@ const steps = (onDownload: OnDownloadFn, onCopy: OnCopyFn) => [
 ];
 
 export const ShareWebsiteFlyout: FC<Props> = ({
-  onCopy,
-  onDownload,
   onClose,
   unsupportedRenderers,
+  renderedWorkpad,
 }) => {
+  const notifyService = useNotifyService();
```

Comment:
Do we want to bring these hooks  that are creating callbacks into the component directly like this, or try to keep them in the container component and passing them as props into the component?

Reply:
Good catch.  I'll address this in my next PR holistically.